lib/plugins/features/movies/controller.js

```diff
@@ -7,3 +7,35 @@ exports.create = async (payload) => {
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.findAll = async (query = {}) => {
+  let title = query.title; //string with or without %, % triggers fuzzy search
+
+  const release_year = query.release_year;
+  const start_year = query.start_year; //exact year or year,year indicating an inclusive range
+  const end_year = query.end_year; //exact year or year,year indicating an inclusive range
```

Comment:
Just a suggestion, I know you have comments here but maybe destructure this and include a comment on another line like this:

```node
// start_year and end_year are an exact year or year,year indicating an inclusive range
const { release_year, start_year, end_year } = query;
```